.pylintrc

```diff
@@ -42,7 +42,7 @@ load-plugins=
 # W0201 : *Attribute %r defined outside __init__*. Because we instanciate object with properties dict
 # C0330 : *Wrong %s indentation%s%s.* Conflict with pep8
 # E0203 : *Access to member %r before its definition line %s*. Because we instanciate object with properties dict
-disable=C1001,W0201,W0212,I0011,W0511,C0330,E0203
+disable=C1001,W0201,W0212,I0011,W0511,C0330,E0203,R0801
```

Comment:
Not sure it's a good idea disable check duplicated code, because it force us to factorise the code